woocommerce/woocommerce
# Store Alerts: Update widths to match main body

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Partial fix for https://github.com/woocommerce/woocommerce/issues/47034



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:


1. Launch a new JN site and skip profiler
2. SSH into the site and force a store alert to appear by running `wp option set woocommerce_db_version 8.9.0`
3. Visit wc-admin pages and legacy pages to see the store alert widths appropriate. Be sure to vary the window size to make sure its responsive.
4. Now set the homescreen view to two columns  by hiding the setup list and refreshing the page. The width of the alert should match the two columns.


Tip: Use `wp option set woocommerce_default_homepage_layout single_column` to reset to single column


### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

Update Store Alert widths to match main body

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
